src/components/init-modal/style.scss

```diff
@@ -318,12 +318,22 @@
 					border-color: var( --wp-admin-theme-color );
 				}
 
+				&:hover {
+					.block-editor-block-preview__container {
+						margin-top: -25%;
+					}
+				}
+
 				.block-editor-block-preview__container,
 				.block-editor-block-preview__content-iframe {
 					cursor: pointer;
 					position: absolute;
 					top: 0;
 				}
+
+				.block-editor-block-preview__container {
+					transition: margin 1s ease-in-out;
```

Comment:
Non-blocking: Personal preference, maybe, but I think it looks better with a 0.5s duration—WDYT?

Reply:
Good idea. https://github.com/Automattic/newspack-newsletters/pull/801/commits/1d780e373e3a00a74ee3363b9c0d961ee35dfdd1